modules/appeals_api/config/schemas/v2/200996.json

```diff
@@ -134,6 +134,12 @@
     "hlrCreateAddress": {
       "type": "object",
       "properties": {
+        "addressLine1":            { "type": "string" },
+        "addressLine2":            { "type": "string" },
+        "addressLine3":            { "type": "string" },
+        "city":                    { "type": "string" },
+        "stateCode":               { "$ref": "#/definitions/hlrCreateStateCode" },
+        "countryName":             { "type": "string" },
```

Comment:
We should probably keep this as `countryCodeISO2` and ensure it only allows two characters, since we likely don't want to do the translation from a roughly-arbitrary string into an [ISO 2-character country code](https://www.nationsonline.org/oneworld/country_code_list.htm), so let's have our consumers pass in the code instead.

If we need `countryName` in the future, we can update to take that in without the need to update the API version.

Reply:
K!